front/client/store/modules/categoryItem.ts

```diff
@@ -15,6 +15,21 @@ const actions: ActionTree<CategoryItemState, any> = {
   async loadCategoryItem({ commit }: ActionContext<CategoryItemState, any>): Promise<any> {
     const data: ICategoryItem[] = await CategoryItemService.getCategoryItem()
     commit('setCategoryItem', data)
+  },
+  async createItem({ dispatch }, { category, name }: ICategoryItem['items']) {
```

Comment:
Wrong type, it should be just "ICategoryItem"

Reply:
fixed